Nos/Models/ReportCategory.swift

```diff
@@ -245,7 +244,9 @@ let categories = [
     ReportCategoryType.other,
 ]
 
-let selectableCategories = [
+let authorCategories = allCategories
```

Comment:
I'm looking at the description for [this ticket](https://github.com/orgs/planetary-social/projects/11/views/1?filterQuery=moderation&pane=issue&itemId=60848564) and the categories listed there are the same as `noteCategories` here. I guess that is a separate ticket and assigned to @joshuatbrown, just pointing it out I guess that I think that ticket could be a one line change if branched off this PR.